text/0000-sbom.md

```diff
@@ -0,0 +1,151 @@
+
+For merging the CycloneDX `bom` files, the `lifecycle` binaries could replicate or use tooling from [CycloneDX-cli](https://github.com/CycloneDX/cyclonedx-cli) which has a merge operation. The only additional piece of information that the `lifecycle` binaries would inject are `CycloneDX` [`metadata`](https://cyclonedx.org/use-cases/#properties--name-value-store) the following property keys -
+
+- `io.buildpacks.bom.buildpack.id` - Buildpack ID for the buildpack that created the BOM
+- `io.buildpacks.bom.layer.name` (Optional) - Set to the name of the layer if the `bom` was associated with a specific layer. 
+
+The `lifecycle` binaries will put the `bom` files from layers set to `launch = true` inside the `/layers/config/sbom/launch/<buildpack-id>/<layer-id>/bom.<ext>` hierarchy otherwise it will put them in `/layers/config/sbom/build/<buildpack-id>/<layer-id>/bom.<ext>`.
+
+## BOM Restoration
+
+During the export phase, the `lifecycle` binaries would be responsible for putting all the `/layers/config/sbom/launch/` directory in the application image as a separate layer and storing its diffIDs in a label for restoring launch layer `bom` files during a rebuild. During a rebuild, it will use previous app image to restore the layer level `launch bom` files.
+
+For non-launch layers, it can just store the files alongside the layer itself and the lifecycle of these `bom` files will be tied to the layer cache lifecycle itself. The `/layers/config/sbom/build/` directory will be an ephemeral directory that is only used during a single build as a convenient place to store all the `build` `bom`s for the platform to export out and make available to the users if needed.
+
+# Backwards compatibility
+
+Since we will not be merging the new formats with the legacy CNB format, we can just output the merged CNB formatted SBOM as a separate file.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This RFC adds a fair bit of complexity to the lifecycle and makes it responsible for more than just buildpack orchestration and container/layer assembly.
+
+# Alternatives
+[alternatives]: #alternatives
+
+Only use one SBOM format like `CycloneDX` or `SPDX` and bake it into the lifecycle.
+
+
+# Prior Art
+[prior-art]: #prior-art
+
+- [#RFC 53](https://github.com/buildpacks/rfcs/blob/main/text/0053-decouple-buildpack-plan-and-bom.md)
+- [#RFC 87](https://github.com/buildpacks/rfcs/blob/main/text/0087-bom-in-layer-metadata.md)
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+- Interactions with the stack SBOM and how to represent that and merge it with the Buildpack SBOM.
+- Restore/export logic for SBOM files. This will be covered in a subsequent RFC.
```

Comment:
With the recent changes from talking with @ekcasey, is this still true @samj1912?

Reply:
Nope - should remove that.